ui/gfxr_vulkan_command_filter_proxy_model.h

```diff
@@ -29,14 +29,25 @@ class GfxrVulkanCommandFilterProxyModel : public QSortFilterProxyModel
     Q_OBJECT
 
 public:
+    enum FilterMode : uint32_t
+    {
+        kNone,
+        kDrawDispatchOnly,
+        kFilterModeCount
+    };
+
     GfxrVulkanCommandFilterProxyModel(QObject                      *parent = nullptr,
                                       const Dive::CommandHierarchy *command_hierarchy = nullptr);
+    void SetFilter(FilterMode filter_mode);
 
 protected:
     bool filterAcceptsRow(int sourceRow, const QModelIndex &sourceParent) const override;
 
 private:
+    void ApplyNewFilterMode(FilterMode new_mode);
+
     const Dive::CommandHierarchy *m_command_hierarchy;
```

Comment:
just to be consistent. if you prefer to initialize members in the header, better to do the init for all members.

Reply:
Done.